tests/bench_pertensor_fp8_old_vs_new.py

```
@@ -0,0 +1,202 @@
+import gc
+import os
+import subprocess
+import sys
+import time
+
+import torch
+
+sys.stdout.reconfigure(line_buffering=True)
+
+
+def run_benchmark_subprocess(model_path, use_old_path, device_ids,
```

Pylint (4.0.4)

[refactor] Too many positional arguments (8/5)

(R0917)